.drone.yml

```diff
@@ -0,0 +1,87 @@
+workspace:
+  base: /go
+  path: src/github.com/src-d/gitbase-playground
+
+branches: [master]
+
+# ANCHORS
+
+build: &build
+  image: golang:1.10-stretch
+  commands:
+    - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```

Comment:
is there any specific reason to install nvm instead of nodejs directly?
```
curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
sudo apt-get install -y nodejs
```

Reply:
I used nvm to copy the exact same steps we have in `.travis.yml`, but now it's changed to your suggestion in a new commit.